src/components/forms/inputs/TextInput.jsx

```diff
@@ -5,16 +5,19 @@ import InputBase from './InputBase';
 
 @injectIntl
 export default class TextInput extends InputBase {
+
     renderInput() {
-        let placeholder;
+        let placeholder; 
 
         if(this.props.placeholder) {
             placeholder = this.props.intl.formatMessage(
                 { id: this.props.placeholder });
         }
 
+        const maxLength = this.props.maxLength ? this.props.maxLength : null
+
         return (
-            <input type="text" value={ this.props.value }
+            <input type="text" maxLength={ maxLength } value={ this.props.value }
```

Comment:
Would the thing behave differently, with `maxLength={ this.props.maxLength }` here?